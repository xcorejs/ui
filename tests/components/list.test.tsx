import 'jest-styled-components';

import { Link } from '@xcorejs/ui';
import React from 'react';
import renderer from 'react-test-renderer';

test('List component', () => {
  const component = renderer.create(
    <>
      <Link href="/">Burger King burger</Link>
    </>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
