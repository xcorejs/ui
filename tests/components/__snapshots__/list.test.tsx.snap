// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List component 1`] = `
.c0 {
  border-color: #0171b6;
  border-bottom: 1px solid transparent;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  font-size: 1.6rem;
  font-size: 1.6rem;
  line-height: initial;
  color: #0171b6;
  -webkit-transition: color 300ms,border 300ms;
  transition: color 300ms,border 300ms;
  cursor: pointer;
  cursor: pointer;
}

.c0:hover {
  border-bottom: 1px solid transparent;
  color: #0169a9;
}

.c0 * {
  cursor: pointer;
}

<a
  className="c0"
  color="primary"
  cursor="pointer"
  display="inline-flex"
  fontSize="1.6rem"
  href="/"
>
  Burger King burger
</a>
`;
